docs-new/docs/graphql/core/how-it-works/metadata-schema.mdx

```diff
@@ -0,0 +1,192 @@
+---
+description: Hasura metadata catalogue
+keywords:
+  - hasura
+  - docs
+  - metadata catalogue
+  - how it works
+---
+
+import Thumbnail from "@site/src/components/Thumbnail/Thumbnail";
+
+# Metadata catalogue
+
+## What is the metadata catalogue?
+
+The Hasura metadata catalogue is a set of internal tables used to manage
+the state of the database and the GraphQL schema. Hasura GraphQL engine
+uses the data in the catalogue to generate the GraphQL API which then
+can be accessed from different clients.
+
+The Hasura GraphQL engine stores this catalogue in a Postgres metadata
+database _(which is by default the same database from which data is
+served over the GraphQL API if it is Postgres)_. When initialized, the
+Hasura GraphQL engine creates a schema called `hdb_catalog` in the
+metadata database and initializes a few tables under it as described
+below.
+
+## hdb_catalog schema
```

Comment:
*hdb_catalog* needs to be emphazised, Please validate the UI against live docs.

Please update here and all other applicable headings.

Reply:
Sure, so I find out that it was bold not emphasized, so update as per the live UI docs.